### PR TITLE
[chore] added vcr cassettes for one drive files info

### PR DESCRIPTION
**Pull request by Kharonus**
### What?

- vcr cassettes for one drive, especially for the files info query

### Why?

- We want to migrate more queries/commands that do outbound request to use vcr over webmock for more stable and real data testing (fun fact: even found a bug, that was hidden by webmock testing stub data, but uncovered by real cassette data)

### CODE REVIEW

File: modules/storages/spec/support/fixtures/vcr_cassettes/one_drive/files_info_query_one_not_found.yml [hunk 1]
```diff
@@ -0,0 +1,93 @@
...
...RmIjoiOCIsInB1aWQiOiIxMDAzMjAwMkVEMkNBMDBFIiwicmgiOiIwLkFRa0FOcjlFVFZhYndFV0lCN3Z6aHQwRWZ3TUFBQUFBQUFBQXdBQUFBQUFBQUFBSkFLOC4iLCJzY3AiOiJEaXJlY3RvcnkuUmVhZFdyaXRlLkFsbCBGaWxlcy5SZWFkV3JpdGUuQWxsIFNpdGVzLlJlYWQuQWxsIFNpdGVzLlJlYWRXcml0ZS5BbGwgVXNlci5SZWFkIHByb2ZpbGUgb3BlbmlkIGVtYWlsIiwic2lnbmluX3N0YXRlIjpbImttc2kiXSwic3ViIjoiWUFYYjFTaUR3RjNCUHFyN2lXWVhGT2YzMGxKSG9PeGtrYzQ4Z0R3c1Q1TSIsInRlbmFudF9yZWdpb25fc2NvcGUiOiJFVSIsInRpZCI6IjRkNDRiZjM2LTliNTYtNDVjMC04ODA3LWJiZjM4NmRkMDQ3ZiIsInVuaXF1ZV9uYW1lIjoibGl2ZS5jb20jZXNjaHViZXJ0Lm9wQG91dGxvb2suY29tIiwidXRpIjoiM2w0WlVBVXlwRWlqZHo3Z21WUWlBQSIsInZlciI6IjEuMCIsIndpZHMiOlsiODhkOGUzZTMtOGY1NS00YTFlLTk1M2EtOWI5ODk4Yjg4NzZiIiwiMTNiZDFjNzItNmY0YS00ZGNmLTk4NWYtMThkM2I4MGYyMDhhIl0sInhtc19zdCI6eyJzdWIiOiIwblA2VEVfQUhBeFM0dm5ibWFRcDFLZUEyUngzbXNBTXphRFQ4RTJ4RmpJIn0sInhtc190Y2R0IjoxNDA1NTE0ODYxLCJ4bXNfdGRiciI6IkVVIn0.J0z20sVW7ClLpsTnGZRJ_vULwXuXAtESGeQ9BidcFrR_k8Pb3Z8DIlgAMAhzK8glbfHVIk3hAOott7GCkEFZ3cWYw2HDvgeHb7urZMuxvMQ9bIrXS8YJLlUP5ATBdy0Xp42i5VzK4xvSxTaihJJh8Sf97MlQL6JRSnh2CEezBGopBKsUW3W6HsvPsypDltRRDBCW5tqD6cPWjv-GkWp0boWSCcP8UGx7ShTILRjIpN6DTB0MqaMgVh3clwYOEYviIGp7yjD4tl_9sJSqImDTXNipg0taTEcR6b8kyQuNWSgJJ0n_cygR3Lh7LbqNQ1ibnOjvA_FcjeYSiYXRBAlEFQ
```
Comment (akabiru) [inline modules/storages/spec/support/fixtures/vcr_cassettes/one_drive/files_info_query_one_not_found.yml]: Should we add a rule to filter the bearer token? I'm not sure if these are short lived tokens- but perhaps safer to filter out by default?

https://benoittgt.github.io/vcr/#/configuration/filter_sensitive_data

Comment (Kharonus) [reply modules/storages/spec/support/fixtures/vcr_cassettes/one_drive/files_info_query_one_not_found.yml]: done, good idea!

File: spec/support/vcr.rb [hunk 2]
```diff
@@ -35,6 +35,12 @@
   config.before_record do |i|
     i.response.body.force_encoding('UTF-8')
   end
+  config.filter_sensitive_data('<ACCESS_TOKEN>') do
+    ENV.fetch('ONE_DRIVE_TEST_OAUTH_CLIENT_ACCESS_TOKEN', 'MISSING_ONE_DRIVE_TEST_OAUTH_CLIENT_ACCESS_TOKEN')
+  end
+  config.filter_sensitive_data('<ACCESS_TOKEN>') do
+    ENV.fetch('NEXTCLOUD_LOCAL_OAUTH_CLIENT_ACCESS_TOKEN', 'MISSING_NEXTCLOUD_LOCAL_OAUTH_CLIENT_ACCESS_TOKEN')
+  end
```
Comment (akabiru) [inline spec/support/vcr.rb]: 🟠 Nice one, I would add these filters in `modules/storages/spec/spec_helper.rb` - as we can extend module specific VCR configuration there.

E.g. cassette directory is configured per module: https://github.com/opf/openproject/blob/dev/modules/storages/spec/spec_helper.rb#L41

**Review by akabiru — APPROVED** [review]
Looks good to me! 👍🏾  One remark to address please

File: spec/support/vcr.rb (hunk 2 above)
Comment (Kharonus) [reply spec/support/vcr.rb]: good hint, will move

**Pull request merged**